converters/toZigbee.js

```diff
@@ -1242,6 +1242,26 @@ const converters = {
             entity.commandResponse('ssIasAce', 'panelStatusChanged', payload);
         },
     },
+    // set the status of keypad in accordance with the assigned Zigbee standard IAS status'
+    set_status: {
+        key: ['set_status'],
+        convertSet: async (entity, key, value, meta) => {
+            const panelStatus = utils.getKeyByValue(common.panStat, value.panelstatus, undefined);
+            const secondsRemain = value.secondsremain = 0;
```

Comment:
I've never seen this syntax before, what does it do?

Reply:
This one set's the indicator LED's and the beeping tones for the keypad. It's different to arm mode which is simply keeps the keypad mode in sync with the server mode but doesn't actually change any indications on the panel